<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewCategoryDialog</class>
 <widget class="QWidget" name="ViewCategoryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ViewCategoryDialog {
          border-image:url(:/images/theme16x9);
          width: 100px;
          border: none;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="2" column="0" colspan="3">
    <widget class="QPushButton" name="pbBack">
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true">background: transparent; border: none;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>512</width>
        <height>225</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="scrollAreaGrid"/>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QScrollBar" name="verticalScrollBar">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>26</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Выберите понравившуюся работу</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
